relay-general/src/pii/convert.rs

```diff
@@ -445,14 +448,30 @@ THd+9FBxiHLGXNKhG/FRSyREXEt+NyYIf/0cyByc9tNksat794ddUqnLOg0vwSkv
         assert_annotated_snapshot!(data);
     }
 
+    /// Checks that any fields containing an IP-address is scrubbed.
+    /// Even if it doesn't make sense for there to be an IP-address
+    /// such as in the username field.
     #[test]
-    fn test_user_ip_stripped() {
+    fn test_ip_stripped() {
         let mut data = Event::from_value(
             serde_json::json!({
                 "user": {
-                    "username": "secret",
-                    "ip_address": "73.133.27.120",
+                    "username": "73.133.27.120", // should be stripped despite not being "known ip field"
+                    "ip_address": "should be stripped despite lacking ip address",
                     "data": sensitive_vars()
+                },
+                "breadcrumbs": {
+                    "values": [
+                        {
+                            "message": "73.133.27.120",
+                            "data": {
+                                "test_data": "73.133.27.120" // test deep wildcard stripping
```

Comment:
As we talked in the meeting, can we also make sure to test that the IPs are scrubbed also from inside the strings, e.g. 

```json
"test_data": "contains ip's like 73.133.27.120 but also the text"
```

which should result in something like 

```json
"test_data": "contains ip's like [ip] but also the text"

```

or make sure this actually can work?

Reply:
I would create a separate issue for this, as it would also the touch IP scrubbing for users who configured it manually through advanced data scrubbing.